### DISCUSSION TITLE
pull_request.branches doesn't trigger on a private repo

### DISCUSSION THREAD

**Discussion by kirkcola** [community/community#26466] [Actions]
<p>Hi,</p>
<p>I have this action.yaml, I want this action to be triggered when someone in my team create a pull request of merging branches into this sprint branch, but it doesn’t seem to work.</p>
<pre><code>on:
  push:
    branches:
      - develop
  pull_request:
    branches:
      - 'sprint/**'

...
</code></pre>
<p>The repo itself is under a github team plan and it’s private, and teams are commiting directly to it with no forks.</p>

**Comment by Yanjingzhu** [comment, marked as answer]
<p>Hi <a class="mention" href="/kirkcola">@kirkcola</a> , </p>
<p>Is the target branch just named “sprint” ? Or “sprint/04” ? </p>
<p>As you set " <strong>sprint/**</strong>" , “sprint” could not match this filter. </p>
<p>There is a worked example for PR merge to sprint branch: </p>
<ol>
<li>In Sprint branch , workflow yml file is : </li>
</ol>
<p><img src="https://user-images.githubusercontent.com/24806478/181097925-0bf79504-9685-4657-91e3-d0e4d789e85c.png" alt="sprint branch.png" data-base62-sha1="g9wV4TfHx6ZASu9TLbM5UytVRHO" width="490" height="417"></p>
<ol start="2">
<li>The PR is from develop to sprint branch:</li>
</ol>
<p>  </p><div class="lightbox-wrapper"><a class="lightbox" href="https://aws1.discourse-cdn.com/github/original/2X/5/5a716db7f95c9ea092cce5ee2d192e38792c719e.png" data-download-href="/uploads/short-url/cU5Zv8kezMFaWvJWM88KfA46VuC.png?dl=1" title="into sprint.png" rel="nofollow noopener"><img src="https://user-images.githubusercontent.com/24806478/181097926-7b7c6a87-79fc-4242-8c15-85b169014a99.png" alt="into sprint.png" data-base62-sha1="cU5Zv8kezMFaWvJWM88KfA46VuC" width="690" height="145" data-small-upload="https://aws1.discourse-cdn.com/github/optimized/2X/5/5a716db7f95c9ea092cce5ee2d192e38792c719e_2_10x10.png"><div class="meta">
<svg class="fa d-icon d-icon-far-image svg-icon" aria-hidden="true"><use xlink:href="#far-image"></use></svg><span class="filename">into sprint.png</span><span class="informations">719×152 9.99 KB</span><svg class="fa d-icon d-icon-discourse-expand svg-icon" aria-hidden="true"><use xlink:href="#discourse-expand"></use></svg>
</div></a></div>

**Comment by kirkcola** [comment]
<p>Thanks for your reply!</p>
